README.md

```diff
@@ -32,11 +32,12 @@ refreshed from production, which is a common question testers ask.
 
 Branches
 --------
-| Moodle version    | Totara version   | Branch   | PHP  |
-| ----------------- | ---------------- | -------- | ---- |
-| Moodle 2.7 to 3.2 | Totara 2.7 to 11 | VERSION1 | 5.5+ |
-| Moodle 3.3 to 3.9 | Totara 12        | VERSION2 | 7.0+ |
-| Moodle 3.10+      | Totara 13        | VERSION3 | 7.3+ |
+| Moodle version     | Totara version   | Branch            | PHP  |
+|--------------------| ---------------- |-------------------|------|
+| Moodle 2.7 to 3.2  | Totara 2.7 to 11 | VERSION1          | 5.5+ |
+| Moodle 3.3 to 3.9  | Totara 12        | VERSION2          | 7.0+ |
+| Moodle 3.10 to 4.4 | Totara 13        | VERSION3          | 7.3+ |
+| Moodle 4.5+        |                  | MOODLE_405_STABLE | 8.1+ |
```

Comment:
can you reverse this please so the latest branch is at the top and the first one you read at a glance